macos_toggle_airplay.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+osascript <<EOS
+
+on toggle()
```

Comment:
AirPlay Receiver uses port 7000 and blocks the docker container from building and using it.
added this script to easily toggle it on/off